blocks/library/block/edit-panel/index.js

```diff
@@ -20,20 +20,22 @@ class SharedBlockEditPanel extends Component {
 	constructor() {
 		super( ...arguments );
 
-		this.bindTitleRef = this.bindTitleRef.bind( this );
+		this.sharedBlockNameField = createRef();
```

Comment:
I think `titleField` would be a more consistent name for this variable. Note that we call it _title_ and not _name_ elsewhere in the code.

Reply:
OK agree consistency is key, though that's not exactly the shared block "title" but it's how users will name the shared block ;)